PUML.h

```diff
@@ -780,17 +820,11 @@ class PUML
 		m_faces.clear();
 		m_v2e.clear();
 
-		unsigned long cellOffset = m_originalSize[0];
-#ifdef USE_MPI
-		MPI_Scan(MPI_IN_PLACE, &cellOffset, 1, MPI_UNSIGNED_LONG, MPI_SUM, m_comm);
```

Comment:
I don't really understand what's happening here and why we can remove it?

Reply:
`cellOffset` is not needed any more, since I replace `i + cellOffset` with `cellDistributor.localToGlobalId(rank, i)` below.